ruby-challenges.rb

```diff
@@ -14,18 +21,32 @@
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+
+
 album1 = 'Rubber Soul'
+puts album1.delete "a,e,i,o,u"
```

Comment:
You know what to use to delete the vowels. Please create a method with this knowledge.